tox.ini

```diff
@@ -3,7 +3,7 @@ envlist = py27,py36,static,pidiff,docs,revdep-pubtools-pulp
 
 [testenv]
 deps=-rtest-requirements.txt
-commands=pytest -v {posargs}
+commands=pytest -vv {posargs}
```

Comment:
This needed or was it used for local testing?

Reply:
That was for local testing, I forgot to remove it